README.md

```diff
@@ -23,3 +25,11 @@ and install the hook:
 ```sh
 pre-commit install
 ```
+
+## Reformating all xml files
+
+Before setting up xmllint, you can reformat all xml files to match xmllinted output with:
```

Comment:
If you leave the check-xmllint read-only option then please provide instructions for how to fix a problem. If this section is that information (as I suspect it is) then please say that instead of the above text. Here is one possible variation:
```
To reformat all XML files (useful if you run check-xmllint in read-only mode):
```
On the other hand: if you only offer the "check and fix" option then you can remove this section because format-xmllint always always fixes things for you.

Reply:
Dropped, as default is format-xmllint hook.